web/client/epics/mapexport.js

```diff
@@ -13,11 +13,12 @@ const { setControlProperty } = require('../actions/controls');
 
 const { mapSelector } = require('../selectors/map');
 const { layersSelector, groupsSelector } = require('../selectors/layers');
+const { backgroundListSelector } = require('../selectors/backgroundselector');
 const { mapOptionsToSaveSelector } = require('../selectors/mapsave');
 const textSearchConfigSelector = state => state.searchconfig && state.searchconfig.textSearchConfig;
 
 const PersistMap = {
-    mapstore2: (state) => JSON.stringify(MapUtils.saveMapConfiguration(mapSelector(state), layersSelector(state), groupsSelector(state), textSearchConfigSelector(state), mapOptionsToSaveSelector(state)))
+    mapstore2: (state) => JSON.stringify(MapUtils.saveMapConfiguration(mapSelector(state), layersSelector(state), groupsSelector(state), backgroundListSelector(state), textSearchConfigSelector(state), mapOptionsToSaveSelector(state)))
```

Comment:
please add a catch method where you dispatch a localized message, in a notification action, that will tell user something as gone wrong and prevent the app to stuck or throw an error